tasks/task_manager.hh

```diff
@@ -169,15 +168,13 @@ public:
                     std::function<std::optional<Res>(const task_essentials&)> map_finished_children) const {
                 auto shared_holder = co_await _lock.hold_read_lock();
 
-                co_return boost::copy_range<std::vector<Res>>(
-                    boost::adaptors::filter(
-                        boost::join(
+                co_return boost::join(
```

Comment:
Can it be [replaced](https://en.cppreference.com/w/cpp/ranges/join_view) with: `std::ranges::views::join`?

Reply:
`std::views::join` requires copyable ranges. while `boost::adaptors::transformed` typically returns lightweight, non-copyable views. The compatibility is uncertain and complex. Most importantly, this is a distraction from the primary goal of transitioning from `boost::adaptors::filtered` to `std::views::filter`.